scripts/fetch-coreos-qemu-image

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -eu
+
+: ${1?"Usage: $0 <coreos-version>"}
+
+COREOS_VERSION=$1
+COREOS_CHANNEL=${2:-stable}
+IMAGE_PATH=$(pwd)/images/qemu/${COREOS_VERSION}
+KERNEL=coreos_production_qemu.vmlinuz
+USRFS=coreos_production_qemu_usr_image.squashfs
+
+mkdir -p ${IMAGE_PATH}
+cd ${IMAGE_PATH}
+
+# remove old images
+rm -f coreos*
+
+wget http://${COREOS_CHANNEL}.release.core-os.net/amd64-usr/${COREOS_VERSION}/coreos_production_pxe.vmlinuz
+wget http://${COREOS_CHANNEL}.release.core-os.net/amd64-usr/${COREOS_VERSION}/coreos_production_pxe.vmlinuz.sig
+wget http://${COREOS_CHANNEL}.release.core-os.net/amd64-usr/${COREOS_VERSION}/coreos_production_pxe_image.cpio.gz
+wget http://${COREOS_CHANNEL}.release.core-os.net/amd64-usr/${COREOS_VERSION}/coreos_production_pxe_image.cpio.gz.sig
+echo "$COREOS_VERSION" > coreos-version
+
+gpg --verify coreos_production_pxe.vmlinuz.sig
+gpg --verify coreos_production_pxe_image.cpio.gz.sig
```

Comment:
You need to import the coreos key. 

https://coreos.com/os/docs/latest/verify-images.html